managed/api-examples/python-simple/user-management.ipynb

```diff
@@ -0,0 +1,205 @@
+    "\n",
+    "headers = {\n",
+    "  'Content-Type': \"application/json\",\n",
+    "  'X-AUTH-YW-API-TOKEN': f\"{yba_api_key}\"\n",
+    "}\n",
+    "\n",
+    "session = requests.Session()\n",
+    "session.verify = False\n",
+    "session.headers = headers\n",
+    "\n",
+    "pprint(yba_url)\n",
+    "pprint(yba_api_key)\n",
+    "pprint(headers)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get the customer ID\n",
+    "\n",
+    "The customer ID is unique to the YBA platform and is required for the following Platform API calls. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "route = f\"{yba_url}/api/v1/session_info\"\n",
+    "response = session.get(url=route).json()\n",
+    "customer_uuid = response[\"customerUUID\"]\n",
+    "\n",
+    "print(customer_uuid)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## List current users\n",
+    "\n",
+    "List users that can login to the YBA platform.\n",
+    "\n",
+    "ref: https://api-docs.yugabyte.com/docs/yugabyte-platform/18e7f5bab7963-list-all-users"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "route = f\"{yba_url}/api/v1/customers/{customer_uuid}/users\"\n",
+    "response = session.get(url=route).json()\n",
+    "pprint(response)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get details for a specific user\n",
+    "\n",
+    "Choose a user from the above list and get their details. Place the user's UUID in the `user_uuid` variable below.\n",
+    "\n",
+    "ref: https://api-docs.yugabyte.com/docs/yugabyte-platform/c11938226a50a-get-a-user-s-details"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "user_uuid=\"asdfasdf-asdf-asdf-asdf-asdfasdfasdf\" # Replace with your user UUID\n",
+    "\n",
+    "route = f\"{yba_url}/api/v1/customers/{customer_uuid}/users/{user_uuid}\"\n",
+    "response = session.get(url=route).json()\n",
+    "pprint(response)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a user\n",
+    "\n",
+    "Create a new user of the YBA platform.\n",
+    "\n",
+    "ref: https://api-docs.yugabyte.com/docs/yugabyte-platform/b37e8d3b835f1-create-a-user"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "email = \"name@email.tld\" # Replace with user's email\n",
+    "password = \"Yo4rPassw*rd\" # Replace with user's password\n",
+    "confirm_password = \"Yo4rPassw*rd\" # Replace with user's password\n",
+    "role = \"ReadOnly\" # Replace with user's role (ConnectOnly, ReadOnly, BackupAdmin, Admin, SuperAdmin)\n",
```

Comment:
Creating a `SuperAdmin` user is not allowed and will throw an exception. So maybe remove that option, or even better mention that here?

Also, when RBAC is enabled, this can also be filled with any of the custom role names. We could also mention that here?